MathDisplay/DataTypes/AliasDictionary.fs

```diff
@@ -0,0 +1,109 @@
+﻿namespace MathDisplay.DataTypes
+
+open System.Collections.Generic
+
+type AliasDictionary<'Key, 'Value when 'Key : equality and 'Value : equality> private(k2v, v2k) =
+    new() = AliasDictionary(Dictionary<'Key, 'Value>(), Dictionary<'Value, 'Key>())
+    new(valueComparer) = AliasDictionary(Dictionary<_, _>(), Dictionary<_, _>(comparer = valueComparer))
+
+    member __.Add(primaryKey, value) =
+        k2v.Add(primaryKey, value)
+        v2k.Add(value, primaryKey)
+    member this.Add(primaryKey, key1, value) =
+        this.Add(primaryKey, value)
+        k2v.Add(key1, value)
+    member this.Add(primaryKey, key1 : 'Key, key2, value) =
+        this.Add(primaryKey, key1, value)
+        k2v.Add(key2, value)
+    member this.Add(primaryKey, key1, key2, key3, value) =
```

Comment:
nonono

Reply:
Just following the `System.String.Format` overloads here.